framework/src/meshgenerators/SideSetsFromBoundingBoxGenerator.C

```diff
@@ -39,9 +39,10 @@ SideSetsFromBoundingBoxGenerator::validParams()
   params.addRequiredParam<subdomain_id_type>(
       "block_id", "Subdomain id to set for inside/outside the bounding box");
   params.addRequiredParam<std::vector<BoundaryName>>(
-      "boundary_id_old", "Boundary id on specified block within the bounding box to select");
-  params.addRequiredParam<boundary_id_type>(
-      "boundary_id_new", "Boundary id on specified block within the bounding box to assign");
+      "boundaries_old",
+      "The list of boundaries on the specified block within the bounding box to be modified");
+  params.addRequiredParam<BoundaryName>(
+      "boundary_new", "Boundary on specified block within the bounding box to assign");
   params.addParam<bool>("boundary_id_overlap",
```

Comment:
you have to keep the old parameters around, just deprecate them.

Then in the body constructor you have logic to handle the user passing deprecated parameters

Reply:
No downstream apps appeared to have been using them. If this is the case, surely the parameters could and should just be replaced.

Reply:
we dont know if any user was relying on this. Their input files would not be attached to an app. We pretty much have to do this because we claim to be doing soft deprecation